### PR TITLE
Commandlet added to manage PIM configurations

### PR DESCRIPTION
**Pull request by v-zhgup**
## Description
</br>
Added commandlet to module to manage PIM configuration

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.Framework/Helpers/CommandHelper.ps1 [hunk 1]
```diff
@@ -256,6 +256,11 @@ class CommandHelper
             Verb = "Get";
             Noun = "AzSKARMTemplateSecurityStatus";
             ShortName = "GATS";
+        },
+		[CommandDetails]@{
+            Verb = "Set";
+            Noun = "AzSKPIMConfiguration";
```
Comment (VishalHaibatpure) [inline src/AzSK.Framework/Helpers/CommandHelper.ps1]: Discuss cmdlet name with Manish

File: src/AzSK/Framework/Core/PIM/PIMScript.ps1 [hunk 2]
```diff
@@ -0,0 +1,556 @@
+
+#Acquire Access token
+class PIM: CommandBase
+{
+    hidden $APIroot = "https://api.azrbac.mspim.azure.com/api/v2/privilegedAccess/azureResources/";
+    hidden $headerParams = "";
```
Comment (VishalHaibatpure) [inline src/AzSK/Framework/Core/PIM/PIMScript.ps1]: Initialize with [String]::Empty

File: src/AzSK.Framework/Helpers/CommandHelper.ps1 (hunk 1 above)
Comment (v-zhgup) [reply src/AzSK.Framework/Helpers/CommandHelper.ps1]: Done discussion

File: src/AzSK/Framework/Core/PIM/PIMScript.ps1 (hunk 2 above)
Comment (v-zhgup) [reply src/AzSK/Framework/Core/PIM/PIMScript.ps1]: done

**Pull request merged**